@xen-orchestra/web/src/utils/create-xo-store-config.util.ts

```diff
@@ -1,7 +1,8 @@
 import type { XoObject, XoObjectContext, XoObjectType, XoObjectTypeToXoObject } from '@/types/xo-object.type'
 import { restApiConfig } from '@/utils/rest-api-config.util'
 import type { SubscribableStoreConfig } from '@core/types/subscribable-store.type'
-import { useFetch } from '@vueuse/core'
+import type { VoidFunction } from '@core/types/utility.type'
```

Comment:
On this branch, the `VoidFunction` type doesn’t exist. It is related to the change in this [PR](https://github.com/vatesfr/xen-orchestra/pull/7812/files#diff-f6b79f629122b297e7827cf7212fd8549eaf71968637e54a4fdd8e1b1988793a).